src/components/routes/homepage/Events.js

```diff
@@ -55,15 +56,11 @@ export const Events = ({ events, me: { username } }) => {
         <Container className={classes.events}>
           {filteredEvents.map(event => {
             const { id, subscribers } = event;
-            const isEventSubscribed = getSubscribedStatus(
-              subscribers,
-              username
-            );
             return (
```

Comment:
since this block of code is removed, I guess `filteredEvents.map({id, subscribers}) => ...` object de-structuring makes sense here

Reply:
that's a good point, but we are still passing `event` as one of the props to `<MediaCard/>` component tho